frontend/tests/components/explore/exporting-markdown.test.tsx

```
@@ -0,0 +1,718 @@
+import { DropletTile } from "@/components/droplets/droplet-tile";
+import { Droplet } from "@/types";
+
+// Mock URL.createObjectURL and URL.revokeObjectURL
+global.URL.createObjectURL = jest.fn(() => "blob:mock-url");
+global.URL.revokeObjectURL = jest.fn();
+
+// Mock Blob constructor
+global.Blob = jest.fn((content, options) => ({
+  content,
+  options,
+})) as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)